crypto/ake/enclave/src/lib.rs

```diff
@@ -165,6 +181,45 @@ impl<EI: EnclaveIdentity> AkeEnclaveState<EI> {
         }
     }
 
+    /// Constructs a ClientAuthRequest to be sent to an enclave backend.
+    pub fn backend_init(&self, backend_id: ResponderId) -> Result<ClientAuthRequest> {
+        let mut csprng = McRng::default();
+
+        let initiator = Start::new(backend_id.to_string());
+
+        let init_input = ClientInitiate::<X25519, Aes256Gcm, Sha512>::default();
+        let (initiator, auth_request_output) = initiator.try_next(&mut csprng, init_input)?;
+        self.backend_auth_pending.lock()?.put(backend_id, initiator);
+        let client_auth_request_data: Vec<u8> = auth_request_output.into();
+        Ok(client_auth_request_data.into())
+    }
+
+    /// Connect to an enclave backend as a client.
+    pub fn backend_connect(
```

Comment:
Should we add a `backend_close` method a la `peer_close` and `client_close`?

Reply:
IIUC, I'm not sure those methods are really used: https://github.com/mobilecoinfoundation/mobilecoin/search?q=client_close. In this case, I also can't see how it would be used effectively- I guess if a Fog View Store goes down, then we would maybe able to call `backend_close`? I'm not entirely sure, but perhaps we can address in a follow-up PR if we find that we need this method because as of right now I don't see a use case. 

@jcape any thoughts?

Reply:
I think it's fine to leave `backend_close` off the list. If we end up with a bunch of stale sessions in the enclave we can use Eran's LRU to keep track of them, but practically speaking the only reason a session would need to be closed is if the store behind it dies.

Reply:
Sounds good.